_layouts/plugins/katello/documentation.html

```diff
@@ -1,6 +1,6 @@
 ---
 pagename: Plugin Manuals
-versions: [{name: 3.9, display: '3.9'}, {name: 3.8, display: '3.8'}, {name: 3.7, display: '3.7'}, {name: 3.6, display: '3.6'}, {name: 3.5, display: '3.5'}, {name: 3.4, display: '3.4'}, {name: 3.3, display: '3.3'}, {name: 3.2, 'display': '3.2'}, {name: 3.1, display: '3.1'}, {name: 3.0, display: '3.0'}, {name: 2.4, display: '2.4'}, {name: 'nightly', display: 'nightly'}]
+versions: [{name: 3.10, display: '3.10-RC'}, {name: 3.9, display: '3.9'}, {name: 3.8, display: '3.8'}, {name: 3.7, display: '3.7'}, {name: 3.6, display: '3.6'}, {name: 3.5, display: '3.5'}, {name: 3.4, display: '3.4'}, {name: 3.3, display: '3.3'}, {name: 3.2, 'display': '3.2'}, {name: 3.1, display: '3.1'}, {name: 3.0, display: '3.0'}, {name: 2.4, display: '2.4'}, {name: 'nightly', display: 'nightly'}]
```

Comment:
:+1:

Reply:
:-1: This doesn't work because it's a number :( If you try going to the 3.10 docs, you end up at 3.1 docs. I think we need to make all the names strings instead.

Reply:
I think you'll need the same thing in plugins/katello/index.md as well.

Reply:
@akofink so I know for the future, how did you test this? i had to test via eyeball and comparing against the 3.9 PR, and wasn't able to find the issue you hit

Reply:
I use the container image! :) https://github.com/theforeman/theforeman.org#with-docker